sdk/communication/pom.xml

```diff
@@ -40,6 +41,11 @@
           <artifactId>azure-communication-sms</artifactId>
           <version>1.0.0-beta.4</version> <!-- {x-version-update;com.azure:azure-communication-sms;current} -->
         </dependency>
+        <dependency>
+          <groupId>com.azure</groupId>
+          <artifactId>azure-communication-identity</artifactId>
+          <version>1.0.0-beta.4</version> <!-- {x-version-update;com.azure:azure-communication-administration;current} -->
```

Comment:
identity not administration, correct?

Reply:
Ah yes, copy + paste booboo, thank you for catching this!